src/crudeditor-lib/components/ViewSwitcher/index.js

```diff
@@ -18,37 +18,49 @@ import {
 
 import WithAlerts from '../WithAlertsHOC';
 
-const ViewSwitcher = ({ activeViewName, modelDefinition, externalOperations, uiConfig }) => {
-  if (!activeViewName) {
-    return null;
+class ViewSwitcher extends PureComponent {
```

Comment:
Why creating a class instead of a function? It should be a function if there's no state nor instance methods.